Tests/PlayerTests/Player/SeekTests.swift

```diff
@@ -98,25 +98,17 @@ final class SeekTests: TestCase {
         expect(player.time).toAlways(beGreaterThanOrEqualTo(player.seekableTimeRange.start), until: .seconds(1))
     }
 
-    func testTimesDuringSeekAfterTimeRangeEnd() {
-        let player = Player(item: .simple(url: Stream.onDemand.url))
-        expect(player.streamType).toEventually(equal(.onDemand))
-        player.play()
-        player.seek(near(player.seekableTimeRange.end + CMTime(value: 10, timescale: 1)))
-        expect(player.time).toEventually(equal(player.seekableTimeRange.end), timeout: .seconds(1))
```

Comment:
It is not better to keep this test and increase its timeout?

Reply:
One test failing on the CI and it was this one :) Reverted.